ts/features/fci/store/actions/fciSignatureRequest.ts

```diff
@@ -0,0 +1,49 @@
+import { ActionType, createAsyncAction } from "typesafe-actions";
```

Comment:
The name of the file is not coherent with the content.
You can:
- rename it as index.ts and maintain all (suggested if you are not going to add many others actions of flows)
- split the actions in different files with coherent naming